### PR TITLE
Bjorncs/apache commons libraries cleanup

### PR DESCRIPTION
**Pull request by bjorncs**
Part of #8788

### CODE REVIEW

File: container/pom.xml [hunk 1]
```diff
@@ -45,10 +45,6 @@
           <groupId>io.airlift</groupId>
           <artifactId>airline</artifactId>
         </exclusion>
-         <exclusion>
```
Comment (gjoranv) [inline container/pom.xml]: Please run 'mvn dependency:tree' before and after to check for any changes.

File: container-test/pom.xml [hunk 2]
```diff
@@ -38,10 +38,6 @@
       <groupId>commons-collections</groupId>
       <artifactId>commons-collections</artifactId>
     </dependency>
-    <dependency>
```
Comment (gjoranv) [inline container-test/pom.xml]: Please verify with 'mvn dependency:tree'

**Comment by gjoranv** [comment]
Please build the whole vespa repo before these changes and run dependency:tree for:
* container
* container-test
* config-model-fat

Keep the files and repeat the procedure with these changes. Then, compare before and after dependency trees.

File: container/pom.xml (hunk 1 above)
Comment (bjorncs) [reply container/pom.xml]: Nothing there.

File: container-test/pom.xml (hunk 2 above)
Comment (bjorncs) [reply container-test/pom.xml]: It's not there 👍

**Review by gjoranv — APPROVED** [review]

**Pull request merged**